src/database/DataTypes/StocktakeItem.js

```diff
@@ -305,7 +305,16 @@ export class StocktakeItem extends Realm.Object {
    */
   get hasReason() {
```

Comment:
The 4 getters at the end of this file should be above the class methods with the rest of the getters.